### PR TITLE
Create weekly subscription page

### LINKED ISSUE #174: Create an newsletter subscription space on the dashboard
## Description
Create an newsletter subscription space on the dashboard

## Required Files
- Subscribe.py

## User Story
As a user I need to subscribe to relevant updates so I can stay up to date.

### PR DESCRIPTION
**Pull request by annalisev**
Create weekly subscription page for the dashboard, so people can sign up for weekly pdf reports.

Closes #174 .

### CODE REVIEW

**Review by shindym — DISMISSED** [review]
Meets all requirements

File: database/schema.sql [hunk 1]
```diff
@@ -9,10 +9,19 @@ DROP TABLE tag CASCADE;
 DROP TABLE game CASCADE;
 DROP TABLE platform_assignment CASCADE;
 DROP TABLE game_tag_matching CASCADE;
+DROP TABLE subscriber
```
Comment (shindym) [inline database/schema.sql:12]: Is this meant to be in this pull request?

Comment (annalisev) [reply database/schema.sql]: No it's not, good spot I will try to fix this!

Comment (shindym) [reply database/schema.sql]: Looks good to me

**Review by shindym — APPROVED** [review]
Changes requested made!

**Pull request merged**